pkg/resources/scaleway/resources.go

```diff
@@ -0,0 +1,267 @@
+package scaleway
+
+import (
+	"fmt"
+	"k8s.io/kops/pkg/resources"
+	"k8s.io/kops/upup/pkg/fi"
+	"k8s.io/kops/upup/pkg/fi/cloudup/scaleway"
+	"strings"
+
+	domain "github.com/scaleway/scaleway-sdk-go/api/domain/v2beta1"
+	"github.com/scaleway/scaleway-sdk-go/api/instance/v1"
+	"github.com/scaleway/scaleway-sdk-go/api/lb/v1"
+	"github.com/scaleway/scaleway-sdk-go/scw"
+)
+
+const (
+	resourceTypeDNSRecord    = "dns-record"
+	resourceTypeLoadBalancer = "load-balancer"
+	resourceTypeVolume       = "volume"
+	resourceTypeServer       = "server"
+
+	KopsDomainName = "scaleway-terraform.com" //TODO: replace with real domain name later
```

Comment:
yes, please add all the plumbing to have this argument set by the command line. take example from the other vendor